### PR TITLE
fix/openapi-doc

### PR DESCRIPTION
**Pull request by jrouzierinverse**
Description
===========
Fixes the OpenAPI documentation and generation

Impacts
=======
The OpenAPI documentation

### CODE REVIEW

File: docs/api/spec/components/schemas/configsource.yaml [hunk 1]
```diff
@@ -107,6 +107,23 @@ ConfigSource:
...
         type: integer
+      message:
+        description: SMS text message ($pin will be replaced by the PIN number)
+        type: string
+      pin_code_length:
+        description: The amount of digits of the PIN number.
+        type: integer
+      sms_activation_timeout:
+        description: This is the delay given to a guest who registered by sms confirmation
+          to fill the pin code.
```
Comment (cgx) [inline docs/api/spec/components/schemas/configsource.yaml]: Use uppercase letters for *sms* and *pin*.

File: docs/api/spec/components/schemas/configsource.yaml [hunk 2]
```diff
@@ -3103,6 +3125,9 @@ ConfigSource:
         description: The amount of times, the local account can be used after its
           created. 0 means infinite.
         type: integer
+      message:
+        description: SMS text message ($pin will be replaced by the PIN number)
+        type: string
       pin_code_length:
         description: The length of the PIN code to be sent over sms
```
Comment (cgx) [inline docs/api/spec/components/schemas/configsource.yaml]: `s/sms/SMS/`

**Review by cgx — CHANGES_REQUESTED** [review]
ApiUser and ApiUsersList should be deleted.

**Comment by cgx** [comment]
Ready to merge?

**Comment by jrouzierinverse** [comment]
Rebased and ready to merge.

**Review by cgx — APPROVED** [review]

**Pull request merged**